### DISCUSSION TITLE
ctrl+space shortcut is reserved

### DISCUSSION THREAD

**Discussion by itaysk** [community/community#5556] [Projects and Issues]
In the new Projects, `ctrl`+`space` is the shortcut to add a new task, but on MacOS, that is the default shortcut to switch languages

**Comment by blinchk** [comment]
Looks that, there should be `Command`+`Space`, because in the most cases `Command` in macOS is equivalent to `Control` in Windows. But `Command`+`Space` is reserved for Spotlight too.

**Comment by RickCogley** [comment]
Confirming that `ctrl-space` on mac does not add a new task but just switches languages. I use it all the time and came here to mention this.

**Comment by szhu** [comment]
I wanted to leave feedback about `Ctrl+Space` for a different reason -- it's just seems like a weird shortcut for adding a new item?

I think that `modifier+Enter` would be more intuitive for adding a new row. In Airtable it's `Shift+Enter` or just `Enter` works too. I think `Cmd/Ctrl-Enter` is better than both of these, since `Shift+Enter`in some other apps means "insert a line break", and `Enter` would cause a lot of accidentally-entered blank items.

**Comment by drone1** [comment]
For me in Windows 10, ``CTRL+SPACE`` switches virtual desktops. And I never set that shortcut. 

Seems like at the very least this should be a shortcut that can be overridden, as a starting point, with just a few options to select from.

Personally I'd want something like ``SPACE`` or ``SPACE SPACE`` since every other combo of using spacebar should be reserved for the operating system if there is no way to prevent propagation if Github Projects handles it.

**Comment by mazgi** [comment]
A reference on macOS is this.
> You can also press Option-Control-Space bar to select the next input source in the Input menu, or Control-Space bar to select the previous input source.

https://support.apple.com/en-by/guide/mac-help/mchlp1406/12.0/mac/12.0
